uploader/main.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+const version = process.env.NW_VERSION;
+
+const download = require("./download");
+const upload = require("./upload");
+const packageUpdater = require("./package-updater");
+
+const downloads = download(version);
+
+const uploads = downloads.then(upload);
+
+uploads
+  .then(function(uploads) {
+    const upload = uploads[0];
```

Comment:
Have you validated that the versions and hashes are the same for the different platforms? Obviously you can only execute one in the module, but it would be worth checking a few manually to have confidence they match.

Reply:
They're not. Which is annoying. The reason I wanted to add the hashes as metadata was if there was a conflict in version numbers it would be easier to resolve.

Reply:
Would it be better to take this out, then?

Reply:
Probably as there is no way to reliably add it.